Kickstarter-iOS/Views/Cells/SettingsPrivacySwitchCell.swift

```diff
@@ -0,0 +1,79 @@
+import Foundation
+import Library
+import Prelude
+
+protocol SettingsPrivacySwitchCellDelegate: class {
+  func privacySettingsSwitchCell(_ cell: SettingsPrivacySwitchCell,
+                                 didTogglePrivacySwitch on: Bool)
+}
+
+final class SettingsPrivacySwitchCell: UITableViewCell, ValueCell, NibLoading {
+  @IBOutlet fileprivate weak var lineLayer: UIView!
```

Comment:
Being a little picky, throughout the app, we name `lineLayer`  by `separatorView`, since it simulates the UITableView separator. I would rename the property for consistency.